index.bs

```diff
@@ -28,6 +28,72 @@ Blah blah blah.
 
 This specification depends on the Infra standard. [[!INFRA]]
 
+<h2 id="deployed-mitigations">Deployed Mitigations</h2>
+
+Some browsers have deployed and announced protections against
+navigational-tracking. This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses an algorithmic approach to combat navigational-tracking. Safari
+labels a site as having navigational-based cross-site tracking capability if
+the following criteria are met:
+
+<ul>
+  <li>The site has not served content as a third-party.</li>
```

Comment:
I want to double-check that "third-party" here is correct, as opposed to "first-party".

Reply:
I believe this is correct, though would welcome a correct from WebKit folks if I got it wrong.

Its taken from https://webkit.org/blog/8311/intelligent-tracking-prevention-2-0/

> ITP 2.0 has the ability to detect when a domain is solely used as a “first party bounce tracker,” meaning that it is never used as a third party content provider but tracks the user purely through navigational redirects.

Though, tbh, its not 100% clear to me if this is descriptive, or part of the algorithm